CHANGELOG.md

```diff
@@ -1,5 +1,75 @@
 # [Versions](https://mui.com/versions/)
 
+## v5.15.12
+
+<!-- generated comparing v5.15.11..master -->
+
+_Mar 4, 2024_
+
+A big thanks to the 16 contributors who made this release possible.
+
+🎨 This is the first release of `pigmentcss` ✨
```

Comment:
```suggestion
🚀 The first public release of Pigment CSS, a zero-runtime CSS-in-JS library.
```

Reply:
```suggestion
🚀 The alpha release of Pigment CSS, a zero-runtime CSS-in-JS library.
```

Building off of Jun's suggestion — I think it's worth mentioning it is an _alpha_ release. "First public release" gives me the impression that it's stable and it could've been tested in private before by selected users (not the case) 😬